package.json

```diff
@@ -42,7 +42,7 @@
     "svchost": "0.1.x",
     "svcinstall": "^0.2.1",
     "validator": "~3.11.0",
-    "vso-node-api": "^0.3.0",
+    "vso-node-api": "^0.3.5",
```

Comment:
This is an unneeded change.  That's what the carrot is for - essentially means 0.3.x.  As a matter of fact, this would be worse because it would prevent it from picking up the bug fixes in the 0.3.6 api